test/e2e/models/CodeServer.ts

```diff
@@ -128,6 +128,8 @@ export class CodeServer {
         path.join(dir, "extensions"),
         "--auth",
         "none",
+        // The last argument is the workspace to open.
+        ...(this.args.includes("--ignore-last-opened") ? [] : [dir]),
```

Comment:
This change allows us to _not_ include the test workspace dir when we pass the `--ignore-last-opened` flag.